console/web3ext/web3ext.go

```diff
@@ -510,6 +510,10 @@ web3._extend({
 			name: 'spamThrottlerConfig',
 			getter: 'admin_spamThrottlerConfig'
 		}),
+		new web3._extend.Property({
```

Comment:
if it is Property, admin_nodeconfig format seems to be more contextual, 
or changes API as web3._extend.Method

Reply:
with this, i think java console is expected to show a function in the form of getGetNodeConfig

Reply:
Thank you. Renamed to `nodeConfig`.